engine/src/main/java/org/terasology/engine/subsystem/common/ConfigurationSubsystem.java

```diff
@@ -46,6 +52,15 @@ public void preInitialise(Context rootContext) {
         config = new Config();
         config.load();
 
+        FlexibleConfigManager flexibleConfigManager = new FlexibleConfigManagerImpl();
+        rootContext.put(FlexibleConfigManager.class, flexibleConfigManager);
+
+        FlexibleConfig renderingFlexibleConfig = new FlexibleConfigImpl();
```

Comment:
Hmmm. I'm thinking that there might be reasons to split the existing rendering config in two separate configs, one for generic, always needed settings, i.e. resolution, and another for all those settings that kick in only in-game. 

The only problem I see is that this would mean some settings become visible to the user only in-game, which is not quite the typical behavior of settings: you can normally tweak them before you enter the game proper. 

On the other hand, given what @vampcat and I have been discussing about render nodes providing their own settings to the RenderingFC, at this stage we only instantiate those nodes when the renderer is alive, which leads anyway to this behavior.

Reply:
I'm definitely up for the split.

And I do think the behaviour of not showing settings in the main menu is the desired one, at least in the long run.

Reply:
<sigh> I don't see how else we can do it admittedly. But it will be a bit unusual I feel. Perhaps on loading/creating a game we'll need a brief alert, fading in/out near the bottom of the screen, informing the user that additional settings have become available.

Reply:
Yeah, the alert seems like a logical idea. That should be easily doable via NUI I think.

( To be fair, since I haven't dived into it too much, I might be totally and horribly wrong :D )